build/build_private_docker.sh

```diff
@@ -32,6 +29,22 @@ docker_image="rmi_pacta_private"
 tag="latest"
 local_tag="$docker_image":"$tag"
 
+# check if az cli is installed
+az_cmd="$(command -v az)"
+if [ -z "${az_cmd}" ]; then
+  echo "Azure CLI not found. Please install it and try again."
+  exit 1
+fi
+
+# check if logged in to az
+az_account="$(az account show -o json)"
+if [ -z "${az_account}" ]; then
+  echo "Please login to Azure using 'az login' and try again."
+  exit 1
+fi
+
+az acr login --name transitionmonitordockerregistry
```

Comment:
Good point. The login is idempotent, so I don't see any issue with calling it. Will add to other script.

Reply:
The `tag-and-push` script already had an `az acr login` call, but it was wrapped in the verbosity check. 

fixed that and parameterized the `acr login` rather than hardcoding in the `build_private_image` script.